src/pages/subjects/science/biology/index.jsx

```diff
@@ -2,12 +2,13 @@ import React from "react"
 
 import Layout from "../../../../components/layout"
 import SEO from "../../../../components/seo"
-import { H1, H6, H3 } from "../../../../components/EasyText"
+import { H1, H2, H3, H4, P } from "../../../../components/EasyText"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['H4' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)